app/charts/shared/legend-color.tsx

```diff
@@ -384,30 +328,29 @@ const LegendColorContent = ({
                     )}
                   </Typography>
                 ) : null}
-                {colorValues.map((d, i) => {
-                  const label = getLabel(d);
+                {colorValues.map((value) => {
+                  const label = getLabel(value);
                   const active = !activeInteractiveFilters.has(label);
 
                   return (
                     <LegendItem
-                      key={`${d}_${i}`}
-                      item={label}
-                      color={getColor(d)}
+                      key={value}
```

Comment:
I wonder why previously we had also i ? Was it because some different values could have the same name ? (canton Zürich and City Zürich?) Perhaps i is necessary ?

Reply:
I checked and in fact you're right, what tricked me here is that we have `value` and `label` that sometimes are both labels, I expected the value to be an iri. I will correct this, thanks 👍